projects/packages/jetpack-mu-wpcom/src/features/wpcom-options-general/options-general.ts

```diff
@@ -55,10 +55,30 @@ const _wpcomBuildSiteUrl = ( fragment: DocumentFragment ) => {
 		const homeUrlLabel = __( 'Site Address (URL)', 'jetpack-mu-wpcom' );
 		const homeUrlRow = document.createElement( 'tr' );
 		homeUrlRow.innerHTML = `
-					<th scope="row"><label for="home">${ homeUrlLabel }</label></th>
-					<td><input type="url" id="home" value="${ window.wpcomSiteUrl.homeUrl }" class="regular-text code disabled" disabled="disabled" /></td>
-			`;
+			<th scope="row"><label for="home">${ homeUrlLabel }</label></th>
+			<td>
+				<input type="url" id="home" value="${
+					window.wpcomSiteUrl.homeUrl
+				}" class="regular-text code disabled" disabled="disabled" />
+				<button type="button" class="button wpcom-add-custom-address-button">
+					&plus; ${ __( 'Add custom address', 'jetpack-mu-wpcom' ) }
```

Comment:
Split to re-use https://translate.wordpress.com/projects/wpcom/es/default?filters%5Bterm%5D=Add+custom+address

Also tried using dashicon here but it didn't look as good as a standard plus sign.

Reply:
Oh it's blocked by context anyway. Ah well.